SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/DeploymentLauncher/DeploymentLauncher.cs

```diff
@@ -26,8 +26,15 @@ internal class DeploymentLauncher
         private const string CHINA_ENDPOINT_URL = "platform.api.spatialoschina.com";
         private const int CHINA_ENDPOINT_PORT = 443;
 
+        // TODO: UNR-2793 - This will only work on windows
```

Comment:
this seems like it could be very easily fixed for mac as well. My main concern is that we have now 2 people in china working on Mac. While they usually use VPN afaik, this could still cause issues if they test the china infrastructure.

Reply:
Perhaps we can pair on this and we can add in a fix for mac.

Reply:
sure :) I'm currently checking with the unity team on how they generate the path as they need to implement the same thing.

Reply:
I just realized that this for launching deployments which won't work on MacOS, because we can't build linux workers on mac. Might be good to add that as a comment somewhere in that flow and then we can close the related ticket. :)